decidim-core/lib/decidim/attributes/localized_date.rb

```diff
@@ -10,7 +10,7 @@ def coerce(value)
 
         Date.strptime(value, I18n.t("date.formats.decidim_short"))
       rescue ArgumentError
-        nil
+        super
```

Comment:
I think this is not working the same way in Virtus, so we need to make some changes here to make it work with Virtus.

Reply:
Should be fixed by 8e35d4410e8871c6ca047b2776cfa3e23deb859d.